helm/hpcc/templates/_helpers.tpl

```diff
@@ -174,26 +193,36 @@ If any storage planes are defined that name pvcs they will be mounted
   {{- end }}
  {{- end }}
 {{- end }}
+{{- end -}}
+
+{{/*
+Add data volume mount
+If any storage planes are defined that name pvcs they will be mounted
+*/}}
+{{- define "hpcc.addDataVolumeMount" -}}
```

Comment:
not new really, but since [can] mount multiple and calls addVolumeMounts now, this should be Mounts too really